src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -27,16 +27,23 @@ object WorkspaceService {
 
 class WorkspaceService(workspaceDAO: WorkspaceDAO) extends Actor {
   override def receive = {
-    case SaveWorkspace(workspace) => saveWorkspace(workspace)
+    case SaveWorkspace(workspace, rootUri) => saveWorkspace(workspace, rootUri)
     case ListWorkspaces => listWorkspaces()
   }
 
-  def saveWorkspace(workspace: Workspace): Unit = {
+  def saveWorkspace(workspace: Workspace, rootUri: Uri): Unit = {
+//    workspaceDAO.load(workspace.namespace, workspace.name) match {
```

Comment:
this commented out block will need to be uncommented once Hussein's clone code is in